configuration.rst

```diff
@@ -333,13 +331,13 @@ configuration file using a special syntax: wrap the parameter name in two ``%``
 
         // config/packages/some_package.php
         namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+        use function Symfony\Component\DependencyInjection\Loader\Configurator\param;
 
         return static function (ContainerConfigurator $containerConfigurator) {
             $containerConfigurator->extension('some_package', [
-                // any string surrounded by two % is replaced by that parameter value
```

Comment:
Keep comment for `%%` case ? Like for xml and yaml

Reply:
Two counter-arguments:
* (More important): It's explained *right above* the code block:
    > wrap the parameter name in two % (e.g. %app.admin_email%):
* (Less important): In PHP, `env()` should be shown as preferred way (IMO).